registry.json

```diff
@@ -4305,5 +4305,19 @@
     "telegram": "EnvyStakePool",
     "operatorName": "Sean",
     "github": "EtherealStakePool"
+  },
+  {
+    "ticker": "SPCP",
+    "poolId": "112bd1da11c5c23e1171505fbbb81c8802d410de9e64373f530723c1",
+    "social": {
+      "website": "https://informatik-spezialist.ch",
+      "twitter": "PKrahemann",
+      "facebook": "https://www.facebook.com/adminInformatik",
+      "instagram": "swiss_premium_cardano_pool",
+      "youtube": ""
+    },
+    "telegram": "swiss_cardano_community",
```

Comment:
Please change this to the TG handle of an individual and not a group or channel

Reply:
@Bez625 
Telegram handle was adjusted.
Greets